src/x11-calc.sh.in

```diff
@@ -119,7 +120,8 @@ _launch() {
          _errmsg="Please install missing fonts in $_fonts."
          ;;
       $ENOENT)
-         _errmsg="Missing ROM file!"
+         _errmsg="`echo "$OPTIONS" | awk -F '-r ' '{ print $2 }' | awk -F '-. ' '{ print $1 }'`"
```

Comment:
A multi line error message works fine in a message box but not so well on the command line, particularly if the output is being redirected to a log file (which I admit is unlikely in this case).  Perhaps define _errmsg for the short message, and _errhelp with a longer explanation to use if zenity is available ?

Reply:
Not sure about the best way to handle, as user on cmdline would aslo appreciate having a more detailed clue on how to fix the error.
Thinking of which... launcher should probably send error messages to `>&2` and return error code on exit.

Reply:
Tried some (interim) trick: thoughts?

Reply:
added double-space to new-line conversion to allow expanded messages both in cmdline and zenity.